test/addons/submariner/start

```diff
@@ -60,6 +60,7 @@ def join_cluster(cluster, broker_info):
         clusterid=cluster,
         cable_driver="vxlan",
         version=VERSION,
+        check_broker_certificate=False,
```

Comment:
Do we want to do this only on macOS? It would be nice to detect real issues with certificates when we can.